fcrepo-http-commons/src/main/java/org/fcrepo/http/commons/session/HttpSession.java

```diff
@@ -27,32 +27,25 @@
 
     private boolean batch = false;
```

Comment:
How will the short-lived and long transactions be distinguished? Using a boolean property in HttpSession?

Reply:
Yes - it is part of the request's state and not part of the transaction's persistent state.  The transaction is short-lived if one of the following two conditions are true: 

1. If there is neither an "atomic-id" nor an "atomic-start" in the http headers
2. If there is a "atomic-id" and a "atomic-commit" in the http headers
3. If there is a "atomic-start" and "atomic-commit"  in the http headers.

Reply:
Decision from meeting: isShortLived() can be part of FedoraTransaction.